### PR TITLE
Add Hadoop groovy script

### PR DESCRIPTION
**Pull request by Mrod1598**

### CODE REVIEW

File: hadoop.groovy [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+def beanJVMMetrics = otel.mbean("Hadoop:service=NameNode,name=JvmMetrics")
+otel.instrument(beanJVMMetrics, "hadoop.jvmmetrics.threads.blocked", "Current number of BLOCKED threads", "1", "ThreadsBlocked", otel.&longValueObserver)
+otel.instrument(beanJVMMetrics, "hadoop.jvmmetrics.threads.new", "Current number of NEW threads", "1", "ThreadsNew", otel.&longValueObserver)
+otel.instrument(beanJVMMetrics, "hadoop.jvmmetrics.threads.runnable", "Current number of RUNNABLE threads", "1", "ThreadsRunnable", otel.&longValueObserver)
+otel.instrument(beanJVMMetrics, "hadoop.jvmmetrics.threads.timed_waiting", "Current number of TIMED_WAITING threads", "1", "ThreadsTimedWaiting", otel.&longValueObserver)
+otel.instrument(beanJVMMetrics, "hadoop.jvmmetrics.threads.terminated", "Current number of TERMINATED threads", "1", "ThreadsTerminated", otel.&longValueObserver)
+otel.instrument(beanJVMMetrics, "hadoop.jvmmetrics.threads.waiting", "Current number of WAITING threads", "1", "ThreadsWaiting", otel.&longValueObserver)
+otel.instrument(beanJVMMetrics, "hadoop.jvmmetrics.gccount", "Total GC count", "1", "GcCount", otel.&longValueObserver)
```
Comment (dehaansa) [inline hadoop.groovy:16]: Should this be a sum?

File: hadoop.groovy [hunk 2]
```diff
@@ -0,0 +1,25 @@
+def beanNameNode = otel.mbean("Hadoop:service=NameNode,name=NameNodeInfo")
+otel.instrument(beanNameNode, "hadoop.namenode.free", "Free", "1", "Free", otel.&longValueObserver)
+otel.instrument(beanNameNode, "hadoop.namenode.used", "Used", "1", "Used", otel.&longValueObserver)
+otel.instrument(beanNameNode, "hadoop.namenode.percentblockpoolused", "Percent block pool used", "%", "PercentBlockPoolUsed", otel.&doubleValueObserver)
+otel.instrument(beanNameNode, "hadoop.namenode.percentused", "Percent used", "%", "PercentUsed", otel.&doubleValueObserver)
+otel.instrument(beanNameNode, "hadoop.namenode.percentremaining", "Percent remaining", "%", "PercentRemaining", otel.&doubleValueObserver)
+otel.instrument(beanNameNode, "hadoop.namenode.totalblocks", "total blocks", "1", "TotalBlocks", otel.&longValueObserver)
```
Comment (dehaansa) [inline hadoop.groovy:7]: Should this be a sum?

File: hadoop.groovy [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+def beanFSNamesystem = otel.mbean("Hadoop:service=NameNode,name=FSNamesystem")
+otel.instrument(beanFSNamesystem, "hadoop.fsnamesystem.num_active_clients", "Current number of active clients holding lease", "1", "NumActiveClients", otel.&longValueObserver)
+otel.instrument(beanFSNamesystem, "hadoop.fsnamesystem.total_load", "Current number of connections", "1", "TotalLoad", otel.&longValueObserver)
+otel.instrument(beanFSNamesystem, "hadoop.fsnamesystem.total_files", "Curent number of files", "1", "TotalFiles", otel.&longValueObserver)
+otel.instrument(beanFSNamesystem, "hadoop.fsnamesystem.stale_data_nodes", "Current number of DataNodes marked stale due to delayed heartbeat", "1", "StaleDataNodes", otel.&longValueObserver)
+otel.instrument(beanFSNamesystem, "hadoop.fsnamesystem.blocks_total", "Current number of allocated blocks in the system", "1", "BlocksTotal", otel.&longValueObserver)
+otel.instrument(beanFSNamesystem, "hadoop.fsnamesystem.total_sync_count", "Total number of sync operations performed by edit log", "1", "TotalSyncCount", otel.&longValueObserver)
```
Comment (dehaansa) [inline hadoop.groovy:24]: sum?

Comment (Mrod1598) [reply hadoop.groovy:24]: [here](https://docs.signalfx.com/en/latest/integrations/agent/monitors/collectd-hadoopjmx.html) is where I got it from.

**Pull request merged**